lib/swagger/swaggerTypes.ts

```diff
@@ -18,6 +18,7 @@ import {
   xmsPaths,
   xmsSkipUrlEncoding,
   xNullable,
+  xMsReadonlyRef,
```

Comment:
nit: but all other xms variables use a lower case M.

Reply:
**sigh**

you're absolutely right :D 

Will update.